images/bin/start.sh

```diff
@@ -51,12 +55,17 @@ while getopts ":s:r" opt; do
 done
 
 syncthingHome=/var/syncthing
+if [ $ephemeral -eq 1 ] && [ -f ${syncthingHome}/executed ]; then
+    echo "fialing: syncthing restarted and persistent volumes are not enabled in the okteto manifest. Run 'okteto down' and try again"
```

Comment:
`fialing`?

What's this warning about? Is it based on something the user did?

Reply:
at this point, the `okteto up` sequence will retry from the beginning. if not, `okteto down` must be run